ccxtbt/ccxtbroker.py

```diff
@@ -128,6 +128,10 @@ def __init__(self, broker_mapping=None, debug=False, **kwargs):
         self.currency = self.store.currency
 
         self.positions = collections.defaultdict(Position)
+        
+        positions = self.store.fetch_positions()
+        for market, pos in positions.items():
+            self.positions[market] = Position(size=pos['size'], price=pos['price'])
```

Comment:
Not sure we can add this to __init__ as fetch_positions() is not part of the unified API as far as I can see. 

https://github.com/ccxt/ccxt/wiki/Manual#unified-api

After checking out the code and running with a script or two, it is appearing to cause an issue with BitFinex and Kraken as those exchanges do not appear to support the endpoint. 

Perhaps you can remove it from `__init__()` and then check for open positions in strategies `next()` or `start()` methods?

Reply:
You're right I should probably subclass that part.

Reply:
I'm developing my own exchange and testing with that, which is on the cowboy side I'll admit. It might also be a little too convoluted but I couldn't figure out how to implement my own stores, feeds and brokers on backtrader - the data structures seemed obscure to me. 

I wish there was something like a REST store to which you supply an URL and that specifies what each endpoint had to return, but alas. Maybe that'll be my next step. In the meantime with ccxt I can implement things in bite sized chunks and so far so good.

In the end it'll be a tool to backtest and deploy strategies connected to a homebrew trading platform.